src/day8/re_frame/trace/styles.cljs

```diff
@@ -95,12 +95,12 @@
   flex: 1;
 }
 #--re-frame-trace-- .panel-content-scrollable {
-  margin: 10px 0 0 10px;
-  flex: 1 0 auto;
-  height: 100%;
+  margin: 0 0 10px 5px;
   overflow: auto;
+  flex: 1 1 auto;
```

Comment:
A heads up: this will probably need to be redone in less/sass when https://github.com/Day8/re-frame-trace/pull/60 is merged.